src/test/resources/org/checkstyle/autofix/recipe/finallocalvariable/multiplevariable/InputMultipleVariable.java

```diff
@@ -0,0 +1,25 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="com.puppycrawl.tools.checkstyle.checks.coding.FinalLocalVariableCheck"/>
+  </module>
+</module>
+*/
+
+package org.checkstyle.autofix.recipe.finallocalvariable.multiplevariable;
+
+public class InputMultipleVariable {
+
+    public void multipleDeclarations() {
+
+        int x = 10, y = 20, z = 30;           // 3 violations
```

Comment:
please add a test case when some of the variables are reset in the definition list, e.g.:
```
int a = 1, b = 2, c = 3; // 1 violation
...
a = 7;
b = 8;
```

So in the case above we expect 1 violation for `c` as it can be made `final`, but `a` and `b` shouldn't be final as they are set with different values later in code.